lib/base/process.cpp

```diff
@@ -162,6 +162,10 @@ static Value ProcessSpawnImpl(struct msghdr *msgh, const Dictionary::Ptr& reques
 		}
 #endif /* HAVE_NICE */
 
+		for (int sig = 1; sig <= 31; ++sig) {
+			signal(sig, SIG_DFL);
```

Comment:
The man page for [signal(2)](https://man7.org/linux/man-pages/man2/signal.2.html) strictly discourages its usage.

> WARNING: the behavior of signal() varies across UNIX versions,
       and has also varied historically across different versions of
       Linux.  Avoid its use: use [sigaction(2)](https://man7.org/linux/man-pages/man2/sigaction.2.html) instead.  See Portability
       below

And in the portability section states:
> Portability
       The only portable use of signal() is to set a signal's
       disposition to SIG_DFL or SIG_IGN.  The semantics when using
       signal() to establish a signal handler vary across systems (and
       POSIX.1 explicitly permits this variation); do not use it for
       this purpose.

It also contains some notes at the end of the pages that say:
> NOTES
       The effects of signal() in a multithreaded process are
       unspecified.

Even all you're doing is resetting the signal handlers to their default action, why use it when there's obviously a simple portable counter piece?